catroid/src/main/res/drawable/ic_input_delete.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2020 The Catrobat Team
+  ~ (<http://developer.catrobat.org/credits>)
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
```

Comment:
since this is a vector, please add size in dp as a suffix to the name, `ic_input_delete_18dp`